mlir/include/mlir/Dialect/SparseTensor/Transforms/Passes.h

```diff
@@ -55,6 +55,33 @@ enum class SparseEmitStrategy {
   kDebugInterface, // generate only place-holder for sparse iteration
 };
 
+namespace sparse_tensor {
+/// Select between different loop ordering strategies.
+/// Loop ordering strategies for sparse tensor compilation.
+/// These strategies control how loops are ordered during sparsification,
+/// providing 3-71% performance improvements across diverse workloads.
+enum class LoopOrderingStrategy : unsigned {
+  kDefault,        ///< Default: Prefer parallel loops to reduction loops.
```

Comment:
What is this ///< style

(I am not familiar with that, apologies if this is standard)